### ISSUE TITLE
Wrong coordinates used in ImageGrab.grab() for macOS

### ISSUE THREAD

**Issue by rkrupski** [python-pillow/Pillow#6308]
Encountered the issue after upgrade from Pillow 9.0.1 to 9.1.1 on macOS.

* OS: Mac
* Python: 3.9.5
* Pillow: 9.1.1

The coordinates used for  _-R_ option of _screencapture_ are incorrectly specified in _ImageGrab.py_ (line 36):
```python
 args += ["-R", f"{left},{right},{right-left},{bottom-top}"]
```
The _screencapture -R_ uses Cartesian pixel coordinate system (x, y, width, height), so the proper usage should be as follow:
```python
 args += ["-R", f"{left},{top},{right-left},{bottom-top}"]
```

Original code snippet from Pillow 9.1.1:
```python
def grab(bbox=None, include_layered_windows=False, all_screens=False, xdisplay=None):
    if xdisplay is None:
        if sys.platform == "darwin":
            fh, filepath = tempfile.mkstemp(".png")
            os.close(fh)
            args = ["screencapture"]
            if bbox:
                left, top, right, bottom = bbox
                args += ["-R", f"{left},{right},{right-left},{bottom-top}"]
            subprocess.call(args + ["-x", filepath])
            im = Image.open(filepath)
            im.load()
            os.unlink(filepath)
            if bbox:
                im_resized = im.resize((right - left, bottom - top))
                im.close()
                return im_resized
            return im
        elif sys.platform == "win32":
```

Grabbing cropped images worked fine in 9.0.1, where the code looked as follow:
```python
def grab(bbox=None, include_layered_windows=False, all_screens=False, xdisplay=None):
    if xdisplay is None:
        if sys.platform == "darwin":
            fh, filepath = tempfile.mkstemp(".png")
            os.close(fh)
            subprocess.call(["screencapture", "-x", filepath])
            im = Image.open(filepath)
            im.load()
            os.unlink(filepath)
            if bbox:
                im_cropped = im.crop(bbox)
                im.close()
                return im_cropped
            return im
```

**Comment by nulano** [comment]
Duplicate of #6244.

The fix is merged into the main branch and will be released with Pillow 9.2.0 on 1 July.

**Comment by rkrupski** [comment]
Good to hear that. Thanks!
